lib/structured_log.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module StructuredLog
+  # See https://github.com/tidelift/design-docs/blob/main/2022-02-backend-logging-conventions.md for conventions
```

Comment:
This isn't a Tidelift project so maybe leave this comment out. It might be time to move this class out into a gem so we don't need to copy this for every project.

Reply:
Good call.